xUnique.py

```diff
@@ -217,21 +229,12 @@ def sort_pbxproj(self, sort_pbx_by_file_name=False):
         pbx_flag = False
         last_two = deque([])
 
-        def file_dir_cmp(x, y):
-            if '.' in x:
-                if '.' in y:
-                    return cmp(x, y)
-                else:
-                    return 1
-            else:
-                if '.' in y:
-                    return -1
-                else:
-                    return cmp(x, y)
+        def file_dir_order(x):
+            return ('.' in x, x)
```

Comment:
Your code snippet is correct. But the difference really happend in python 2.7 and python 3.5
So the problem is in python 3 string sort method modification in line 272.